components/calendar/src/astronomy.rs

```diff
@@ -1199,8 +1197,10 @@ impl Astronomical {
     /// Reference lisp code: https://github.com/EdReingold/calendar-code2/blob/9afc1f3/calendar.l#L7068-L7074
     #[allow(clippy::unwrap_used)]
     pub(crate) fn month_length(date: RataDie, location: Location) -> u8 {
-        let moon = Self::phasis_on_or_after(date + 1, location);
-        let prev = Self::phasis_on_or_before(date, location);
+        let lunar_phase =
```

Comment:
suggestion: we should have an intermediate `lunar_phase` function for this.

Reply:
done.